src/test/java/org/folio/rest/impl/HelperUtilsTest.java

```diff
@@ -30,6 +30,8 @@ public class HelperUtilsTest extends TestBase {
   private static final String GROUP_BUDGET_ENDPOINT = getEndpoint(FinanceStorageGroupBudgets.class);
 
   @Test
+  @Disabled("disabled due to incompatibility of jdk17 + jmockit + jacoco"
+    + "remove annotation and check with command: 'mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install' ")
```

Comment:
Disabled due to incompatiblity
reference to commit: https://github.com/folio-org/mod-orders-storage/pull/354/files#r1377421918
![image](https://github.com/folio-org/mod-finance-storage/assets/113523904/fc845a75-5eb6-4f46-b782-4ae1a4d15e55)